app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cd = command.split(" ")
```

Comment:
The variable `cd` is not descriptive. Consider renaming it to something more meaningful, like `command_parts`, to improve code readability and maintainability.